code/game/machinery/teleporter.dm

```diff
@@ -268,10 +268,13 @@
 		to_chat(M, "<span class = 'notice'>The act of teleportation was so smooth, it feels like you didn't move at all.</span>")
 		return 0
 	if (istype(M, /atom/movable))
+		/*
```

Comment:
Using block comments to comment out code isn't recommended, as it tends to fuck up version control systems
For example, that's how we got MEGAMADCRAB